sql-delta-import/src/main/scala/com/razorpay/spark/jdbc/JDBCImport.scala

```diff
@@ -150,25 +150,27 @@ class JDBCImport(
   private def readJDBCSourceInParallel(): DataFrame = {
 
     if (importConfig.splitBy.nonEmpty) {
+      val defaultString="defaultValue"
       val (lower, upper) = spark.read
         .jdbc(buildJdbcUrl, importConfig.boundsSql, jdbcParams)
-        .selectExpr("cast(min as long) min", "max")
-        .as[(Option[Long], Option[Long])]
+        .selectExpr("cast(min as string) min", "cast(max as string) max")
+        .as[(Option[String], Option[String])]
         .take(1)
-        .map { case (a, b) => (a.getOrElse(0L), b.getOrElse(0L)) }
+        .map { case (a, b) => (a.getOrElse(defaultString), b.getOrElse(defaultString)) }
         .head
 
-      spark.read
-        .jdbc(
-          buildJdbcUrl,
-          importConfig.jdbcQuery,
-          importConfig.splitColumn,
-          lower,
-          upper,
-          importConfig.chunks,
-          jdbcParams
-        )
-        .where(s"${importConfig.splitColumn} >= $lower and ${importConfig.splitColumn} <= $upper")
+      spark.read.format("jdbc")
+        .option("url",buildJdbcUrl)
+        .option("dbtable",importConfig.jdbcQuery)
+        .option("partitionColumn",importConfig.splitColumn)
```

Comment:
double check this. split column is different from partition column.

Reply:
<img width="1000" alt="Screenshot 2022-09-09 at 3 45 33 PM" src="https://user-images.githubusercontent.com/112069583/189327985-949bb847-eba5-4786-92c3-7528ad8bb74c.png">
In the earlier code we were using splitColumn from inputs as partitionColumn

Reply:
<img width="1377" alt="Screenshot 2022-09-07 at 12 37 07 PM" src="https://user-images.githubusercontent.com/112069583/189328131-22a682ef-c0ec-487d-9ba7-4a9da86d1f04.png">